app/src/main/java/org/fossasia/openevent/general/order/Order.kt

```diff
@@ -47,5 +47,5 @@ data class Order(
     @Relationship("event")
     var event: EventId? = null,
     @Relationship("attendees")
-    var attendees: List<AttendeeId>? = null
+    var attendees: ArrayList<AttendeeId>? = null
```

Comment:
You made exactly opposite change in a file which has no relation to the attendee view model